### ISSUE TITLE
Reflection: Ng Teng Hui

### ISSUE THREAD

**Issue by AstralAurora** [hanz105/INF2001-Team-Project-P8-3#33]
I've learnt a lot about software engineering and the various software development techniques throughout this project's M1, though mostly I believe I learned about project management and how to work effectively with my team.

One of the most important things I learned was how to use the SDLC to help us with our project work. Because we practised using the waterfall and agile methodologies in our Gantt chart, I believe the knowledge and experience we gained will help us during our future work studies and be useful in our future careers. As requirements can change, using GitHub has taught me the value of teamwork and communication in our projects as well as the need to be flexible. 

As far as I know, software engineering mostly involves various teams working together to accomplish various tasks, not many projects would be a solo endeavour. Therefore, collabourating with my teammates, whom each bring their own skills and, more importantly to me, perspectives to the team. 

In conclusion, this M1 of our project has been an fruitful learning endeavour, equipping me with not only practical experience for a software engineering project, but also various soft skills that are a requirement for a successful career in the future.
